layouts/partials/footer.html

```diff
@@ -53,6 +53,9 @@
                 <a class="footer-policies-link" href="https://www.linuxfoundation.org/legal/privacy-policy">
                     {{ i18n "footer_privacy_policy" }}
                 </a> |
+                <a class="footer-policies-link" href="https://www.linuxfoundation.org/legal/trademark-usage">
+                    {{ i18n "footer_trademarks" }}
```

Comment:
something looks like it's missing on the [netlify preview](https://deploy-preview-14205--preliminary-istio.netlify.app/), the text `Trademarks` does not appear within ` |  | `

Reply:
@GregHanson it looks as I expect it to -- are you seeing something different?

![image](https://github.com/istio/istio.io/assets/2148318/dbb3e713-65fe-4efa-9de5-b7342561fc73)

Reply:
I see it in the netlify preview as well.

Reply:
<img width="582" alt="Screenshot 2023-11-30 at 9 02 16 AM" src="https://github.com/istio/istio.io/assets/19473391/cbec0180-faf6-47aa-a592-a2a6020d8383">

Odd, there must have been some sort of delay.  I see now after refreshing my page